app/pages/single/single.scss

```diff
@@ -1 +1,52 @@
+.single {
+  .post-header {
```

Comment:
is .post-header needed or can we skip the class and replace it with  ion-toolbar?